### PR TITLE
Send task logs to Fluent Bit

### PR DESCRIPTION
**Pull request by branden**
## High-level description

This PR bumps mesos-modules to pull in https://github.com/dcos/dcos-mesos-modules/pull/105, and adds configuration that directs Mesos agents to send task logs to Fluent Bit.

This also simplifies the Fluent Bit build script by moving the logic for defining Fluent Bit's service name to dcos-config.yaml.

I manually tested that task logs are being sent to Fluent Bit. I launched a Marathon app where tasks echo on a loop, configured dcos-fluent-bit.service on an agent to output logs to stdout, and observed task logs in dcos-fluent-bit.service's journal.

## Corresponding DC/OS tickets (obligatory)

- [DCOS-50078](https://jira.mesosphere.com/browse/DCOS-50078) Integrate Mesos task log module changes with DC/OS

## Related tickets (optional)

N/A

## Checklist for all PRs

  - [x] Added a comprehensible changelog entry to `CHANGES.md` or explain why this is not a user-facing change: Fluent Bit isn't yet configured to do anything useful, so this has no user-facing impact.
  - [x] Included a test which will fail if code is reverted but test is not. If there is no test please explain here: Tested manually, see description above.
  - [x] Read the [DC/OS contributing guidelines](https://github.com/dcos/dcos/blob/master/contributing.md)
  - [x] Followed relevant code rules [Rules for Packages and Systemd](https://github.com/dcos/dcos/tree/master/docs)

## Checklist for component/package updates:

If you are changing components or packages in DC/OS (e.g. you are bumping the sha or ref of anything underneath `packages`), then in addition to the above please also include:

  - [x] Change log from the last version integrated (this should be a link to commits for easy verification and review): https://github.com/dcos/dcos-mesos-modules/compare/2ac2fcebe7872da05571078d68207c2d622e3acd...2633cb7135c978f640d08819f2408c541d6dedb2
  - [x] Test Results: [link to CI job test results for component](https://jenkins.mesosphere.com/service/jenkins/job/mesos/job/modules/job/DCOS_OSS_Mesos_Modules-pr/158/)
  - [x] Code Coverage (if available): N/A

### CODE REVIEW

File: packages/fluent-bit/build [hunk 1]
```diff
@@ -14,20 +14,7 @@ popd
...
-
-# Add the agent unit files to the package.
-# Agents and public agents have the same unit file contents.
-for unit in "$agent_unit" "$agent_public_unit"; do
-  mkdir -p $(dirname "$unit")
-  SERVICE="dcos-fluent-bit-agent" envsubst '${SERVICE} ${PKG_PATH}' < "$unit_template" > "$unit"
-done
+# Add systemd unit file.
+unit_file="$PKG_PATH/dcos.target.wants/dcos-fluent-bit.service"
+mkdir -p "$(dirname "$unit_file")"
```
Comment (philipnrmn) [inline packages/fluent-bit/build]: Do these nested quotes not cause a problem?

**Comment by philipnrmn** [comment]
Looks excellent. I had a query about one line of bash but I'm probably wrong.

File: packages/fluent-bit/build (hunk 1 above)
Comment (branden) [reply packages/fluent-bit/build]: It surprised me as well, but they do work as intended:

```
branden@crateria-14359 ~$ foo="/foo/bar"
branden@crateria-14359 ~$ echo "$(dirname "$foo")"
/foo
branden@crateria-14359 ~$
```

I'd normally remove them like I try to do with other surprising bash syntax, but these quotes will help make sure that bash always treats the enclosed value as a single string.

**Comment by branden** [comment]
@mesosphere-mergebot label ready for review

File: packages/fluent-bit/build (hunk 1 above)
Comment (philipnrmn) [reply packages/fluent-bit/build]: bash works in mysterious ways. Thanks!

**Comment by branden** [comment]
@mesosphere-mergebot sync

**Pull request merged**